deep/token.py

```diff
@@ -18,6 +19,9 @@ class DeepTokenGenerator(PasswordResetTokenGenerator):
     class Meta:
         abstract = True
 
+    def make_token(self, instance: models.Model):
```

Comment:
Isn't this definition redundant?

Reply:
This is just for type definition for now.